### DISCUSSION TITLE
what is the meaning of v_num?

### DISCUSSION THREAD

**Discussion by gyuchulm** [acids-ircam/RAVE#191] [Q&A]
What is the meaning of the v_num? End of the epoch?
Is it a meaning of the latent size?

**Comment by jacklion710** [comment]
Perhaps you might find [this answer](https://github.com/Lightning-AI/lightning/issues/1595) helpful.

"_v_num is the version number of the experiment. Depending on the logger, this can be a simple counter that increases everytime you run the trainer_"

It appears to be part of pytorches way of counting every time the training process is run
